README.md

```diff
@@ -38,3 +43,23 @@ to either a Vagrant or web host instance.
     - `vagrant up`
     - This step runs the `ansible` provisioner the first time. If you want to make changes
       to `orchestration`, run `vagrant provision` afterwards to refresh.
+
+## Setting up EXISTING Yurt projects:
+
+### Establishing development environment:
+- Git clone this deployment repository into its own project directory
+    - `mkdir ~/projects`
+    - `git clone https://github.com/yeti/yurt.git ## Makes a directory called "yurt"`
+- Make a new project directory and navigate to it
+    - `mkdir ~/projects/new_proj`
+    - `cd ~/projects/new_proj`
+- Symlink the fabfile to the project directory
+    - `sudo ln -s ~/projects/yurt/fabfile .`
+- Call the `setup.add_settings` fabric method to generate `fabric_settings.py`
+    - `fab setup.add_settings`
+- Edit `fabric_settings.py`, filling in the blank values.
+    - `nano fabric_settings.py`
```

Comment:
Also, if they're setting up an existing project would they have to edit a settings file? Wouldn't this all be setup already by the creator of the project?